test/test_code_utils.py

```diff
@@ -552,6 +553,19 @@ def test_non_dict_in_list(self):
         with self.assertRaises(TypeError):
             content_str(content)
 
+# Test cases for dangerous commands that should be caught by the sanitizer
```

Comment:
Move the test to it's own module in `test/coding/`

Reply:
done 👍